### ISSUE TITLE
parser_error:  false positive in select

### ISSUE THREAD

**Issue by g-back** [abaplint/abaplint#649]
```abap
SELECT kna1~kunnr,
    kna1~name1,
    kna1~ort01,
    kna1~stras,
    kna1~pstlz,
    kna1~stceg
  FROM kna1
  WHERE kna1~ktokd = 'ZAG'
    AND kna1~sperr = ' '
    AND kna1~sperz = ' '
  GROUP BY kunnr, name1, ort01, stras, pstlz, stceg
  INTO TABLE @DATA(mt_selection).
```
is marked with a `parser_error`.  Apparently the `GROUP BY` elements can't be separated by commas. 
Correct behaviour: If the select-from elements are comma-separated, the group-by elements have to be as well.

**Comment by larshp** [comment]
thanks

plan is to have the syntax allow commas, and a rule to check if the SQL is strict SQL
